src/templates/community-champion-profile.js

```diff
@@ -27,16 +82,196 @@ const StyledBreadcrumb = styled(Breadcrumb)`
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const QuoteContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    grid-area: quote;
+    padding: 0 ${size.xsmall};
+    text-align: center;
+`;
+
+const QuoteIconContainer = styled('div')`
+    height: ${QUOTATION_MARKS_ICON_HEIGHT};
+    margin-bottom: ${size.mediumLarge};
+`;
+
+const QuoteIcon = styled('img')`
+    max-height: 100%;
+`;
+
+const QuoteSection = ({ quote }) => (
+    <QuoteContainer>
+        <QuoteIconContainer>
+            <QuoteIcon src={QuotationMarksIcon} alt="Opening quotation marks" />
+        </QuoteIconContainer>
+        <Quote collapse>{quote}</Quote>
+    </QuoteContainer>
+);
+
+const TextContainer = styled('div')`
+    grid-area: text;
+`;
+
+const ImageContainer = styled('div')`
+    display: flex;
+    grid-area: image;
+    justify-content: center;
+    margin-bottom: ${IMAGE_CONTAINER_BOTTOM_MARGIN};
+    @media ${screenSize.upToSmallDesktop} {
+        margin-bottom: ${size.default};
+    }
+`;
+
+const InfoGrid = styled('div')`
+    column-gap: ${size.large};
+    display: grid;
+    grid-template-areas:
+        'occupation languages'
+        'location languages';
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+    margin-bottom: ${INFO_GRID_BOTTOM_MARGIN};
+    row-gap: ${INFO_GRID_ROW_GAP};
+    @media ${screenSize.upToSmallDesktop} {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        margin-bottom: ${size.large};
+    }
+`;
+
+const IconWithTextContainer = styled('div')`
+    align-items: start;
+    column-gap: ${size.xsmall};
+    display: grid;
+    ${({ gridArea }) => (gridArea ? `grid-area: ${gridArea}` : '')};
+    grid-template-columns: ${size.default} ${TEXT_COLUMN_WIDTH};
+    @media ${screenSize.upToSmallDesktop} {
+        grid-template-columns: ${size.default} auto;
+        text-align: center;
+    }
+    @media ${screenSize.upToMedium} {
+        grid-template-columns: ${ICON_MOBILE_WIDTH} auto;
+    }
+`;
+
+const Icon = styled('img')`
+    padding-top: ${size.tiny};
+    width: 100%;
+`;
+
+const IconWithText = ({ gridArea, icon, iconAltText, text }) => (
+    <IconWithTextContainer gridArea={gridArea}>
+        <Icon src={icon} alt={iconAltText} />
+        <P2 collapse>{text}</P2>
+    </IconWithTextContainer>
+);
+
+const Occupation = ({ occupation }) => (
+    <IconWithText
+        gridArea="occupation"
+        icon={BriefcaseIcon}
+        iconAltText="Briefcase"
+        text={occupation}
+    />
+);
+
+const Location = ({ location }) => (
+    <IconWithText
+        gridArea="location"
+        icon={LocationPinIcon}
+        iconAltText="Location pin"
+        text={location}
+    />
+);
+
+const LanguagesTitle = styled(H6)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    font-size: ${fontSize.tiny};
+    font-weight: 500;
+    letter-spacing: 0.1em;
```

Comment:
I don't think this is having the intended effect as it seems pretty squished on the staging link? (Actually I think the staging link is just a bit outdated)

Reply:
ahh my bad, i forgot to update the staging link!